var/spack/repos/builtin/packages/nalu-wind/package.py

```diff
@@ -116,6 +119,11 @@ def cmake_args(self):
         else:
             options.append('-DENABLE_FFTW:BOOL=OFF')
 
+        if '+wind-utils' in spec:
+            options.extend([
+                '-DENABLE_WIND_UTILS=ON'
+            ])
```

Comment:
Use append instead of extend when you only have a single element to add. Can you explicitly disable this in an else clause? Even better would be to use the `define_from_variant` method from the `CMakePackage` base class.